Examples/en/TypeScript/Primitives/Any.ts

```diff
@@ -0,0 +1,33 @@
+// Any is the TypeScript escape clause. You can use any to
+// either declare a section of your code to be dynamic and
+// JavaScript like, or to work around limitations in the
+// type system.
+
+// A good case for any is JSON parsing:
+
+const myObject = JSON.parse("{}");
+
+// Any declares to TypeScript to trust your code as being
+// safe because you know more about it. Even if that is
+// not strictly true. For example, this code would crash:
+
+myObject.x.y.z;
+
+// Using an any gives you the ability to write code closer to
+// original JavaScript with the trade off of type safety.
+
+// Any is considered a top type in type theory, which means
```

Comment:
This is true in TypeScript, but not true in the gradual typing literature.

I think a more accurate way of describing it from a theoretical point of view (which may not be useful in this context) is roughly:

_any is much like a 'type wildcard' which you can replace with any type  (except never)_ to make one type assignable to the other.

_Each call to debug is allowed because I could replace the `any` with the type of the argument to match._
```ts
declare function debug(value: any);
	
debug("a string");
debug(23);
debug({ color: "blue" });
```

_The call to swap is allowed because the argument type can be matched by replacing the first `any` in `pair` with `number` and the second `any` with `string`._
```ts
declare function swap(x: [number, string]): [string, number]
declare const pair: [any, any];
swap(pair)
```

Reply:
Ah yeah, a type wildcard is a great analogy 👍